slatedb/src/db_cache/mod.rs

```diff
@@ -236,6 +236,124 @@ impl CachedEntry {
     }
 }
 
+pub struct SplitCache {
+    // Cache for block data
+    block_cache: Option<Arc<dyn DbCache>>,
+    // Cache for indices and filters
+    meta_cache: Option<Arc<dyn DbCache>>,
+}
+
+impl SplitCache {
+    #[inline(always)]
+    pub fn default_block_capacity() -> u64 {
+        // 512 MB
+        512 * 1024 * 1024
+    }
+
+    #[inline(always)]
+    pub fn default_meta_capacity() -> u64 {
+        // 128 MB
+        128 * 1024 * 1024
+    }
```

Comment:
These two functions could be constants, couldn't they?

```suggestion
  const DEFAULT_BLOCK_CAPACITY: u64 = 512 * 1024 * 1024;
  const DEFAULT_META_CAPACITY: u64 = 128 * 1024 * 1024;
```

Reply:
I personally prefer to stick with the idioms of the language at hand, instead of trying to emulate some other language. I’ve seen more often static values referred as constants than static functions in Rust.

Constants are also automatically inlined, basically achieving what you’re trying to do without having to force the compiler to do it:

> Constants live for the entire lifetime of a program. More specifically, constants in Rust have no fixed address in memory. This is because they’re effectively inlined to each place that they’re used. References to the same constant are not necessarily guaranteed to refer to the same memory address for this reason.